build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts

```diff
@@ -517,7 +517,11 @@ function parseFluidImports(
 }
 
 function isFluidImport(packageName: PackageName): boolean {
-	return packageName.startsWith("@fluid") || unscopedFFPackages.has(packageName);
+	return (
```

Comment:
Nit: probably worth adding some notes here about why the narrow checks are needed.

Reply:
And we'll want to remember if we every deviate from any of those.

Reply:
Added some docs, but I also switched to using a fixed list of known scopes.